tests/auto/settings/tst_settings.cpp

```
@@ -179,59 +179,47 @@ namespace {
 struct BoolSetting {
   const char *name;
   void (*setter)(const bool &);
-  bool (*getter)(const bool &);
+  bool AppSettings::*field;
 };
 
 const BoolSetting boolSettings[] = {
```

Clang (14.0.6)

[error] do not declare C-style arrays, use std::array<> instead

(modernize-avoid-c-arrays,-warnings-as-errors)